doc/ChangeLog

```diff
@@ -1,4 +1,112 @@
 ===============================================================
+Tag name:  ctsm1.0.dev113
+Originator(s):  negins (Negin Sobhani,UCAR/TSS,303-497-1224)
+Date: Tue Sep  8 15:57:49 MDT 2020
+One-line Summary: Some bit-for-bit changes needed for the Perturbed Parameter Ensemble work
+
+Purpose of changes
+------------------
+
+Some BFB fixes including bug fixes needed for PPE simulations. 
+
+Bugs fixed or introduced
+------------------------
+
+Issues fixed (include CTSM Issue #):
+- Resolves ESCOMP/CTSM#1060 (Some code cleanup of luna bug fixes)
+- Resolves ESCOMP/CTSM#1062 (PPE namelist parameter defaults)
+- Resolves ESCOMP/CTSM#1053 (Non-standard namelist parameter)
+- Resolves ESCOMP/CTSM#1055 (Expand potential vcmax:jmax ratio)
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[ ] clm4_5
+
+Notes of particular relevance for users
+---------------------------------------
+
+Caveats for users (e.g., need to interpolate initial conditions):
+cmb_cmplt_fact was a parameter under namelist control which included an array.
```

Comment:
@negin513 one suggestion here. I would explicitly mention that if someone has cmb_cmplt_fact in a user_nl_clm file that worked before -- it won't work now since it's. split into two. It's good to try to think about it from the user's perspective and what will break for them because of the change. The thing here is that a user_nl_clm change that worked before won't anymore because of this. We want to try to explain things from the user perspective on what's different (what's new, what's removed) and what's different, and especially as in this case what would work before -- but won't work anymore because of the change.

Reply:
I think there's three different types of user's that we want to make the ChangeLog useful for. One is for the scientists that just want to know the big picture science that is happening to the model. The next are the User's who are just using the model, and we want to tell them what's changed from their perspective. This section here is primarily targeted to that group.

The next section for developers is primarily targeting you, and Bill and I. But, also could be other developers working in the guts of the code. That section should give notes about the code changes about things like reminding us to clean this up later, that a code section should be refactored, or whatever useful bit of advice we might want to remember later. I'll use it to let you and Bill know about something, but I also use it as a reminder for me as well.